playkit/src/main/java/com/kaltura/playkit/OnMediaInterceptorListener.java

```diff
@@ -0,0 +1,5 @@
+package com.kaltura.playkit;
+
+public interface OnMediaInterceptorListener {
+    void onApplyMediaCompleted();
+}
```

Comment:
This class has to move to Kaltura Player.

Reply:
This interface is used by Plugin/Interceptor to notify that it finishes its work. So Broadpeak plugin will have to add kaltura-player dependency as well.

Reply:
As the interceptors are meaningless for PlayKit, declaring the interfaces in PlayKit is confusing.

Reply:
ok, i will move it to KalturaPlayer